src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Migrations/20240501113006_InitialMigration.cs

```diff
@@ -579,40 +580,67 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 principalTable: "DataSetVersions",
                 principalColumn: "Id");
 
+            var isLocalEnvironment = 
+                Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == Environments.Development;
+
             // Grant privileges on objects created by this resource's database user to the Admin App Service user.
-            var adminAppServiceIdentityName = Environment.GetEnvironmentVariable("AdminAppServiceIdentityName");
-            if (adminAppServiceIdentityName != null)
+            var adminAppServiceRoleName = isLocalEnvironment
+                ? "app_admin" 
+                : Environment.GetEnvironmentVariable("AdminAppServiceIdentityName");
```

Comment:
I wonder, should we just fail initial migration if we're running in Azure and we haven't been supplied the appropriate environment variables for the various users?  Failing to do so will allow the migration to complete successfully but without all our required users set up, at which point we'd have to do manual cleanup.

Reply:
See previous comment about this not being required any more, plus a possible solution for it it's required again.

I've also removed these variables from the bicep template.

Reply:
:+1: